src/Plugin/PluginImplementation.php

```diff
@@ -18,6 +14,13 @@
  * The TYPO3 project - inspiring people to share!
  */
 
+namespace TYPO3\CMS\Composer\Plugin;
+
+/**
+ * This file was taken from the typo3 console plugin package.
+ * (c) Helmut Hummel <info@helhum.io>
+ */
```

Comment:
I'm fine to remove this now. plugin package is deprecated anyway and the canonical implementation is here